### PR TITLE
isympy: marked main function's return type as None

### PR DESCRIPTION
**Pull request by michelececcacci**
#### Release Notes

NO ENTRY

### CODE REVIEW

File: isympy.py [hunk 1]
```diff
@@ -173,7 +173,7 @@
 # DO NOT IMPORT SYMPY HERE! Or the setting of the sympy environment variables
 # by the command line will break.
 
-def main():
+def main() -> None:
     from argparse import ArgumentParser, RawDescriptionHelpFormatter
```
Comment (oscarbenjamin) [inline isympy.py:177]: I don't see the point of making this change.

Comment (michelececcacci) [reply isympy.py:177]: If you feel like this is pointless we can close this.

Comment (oscarbenjamin) [reply isympy.py:177]: Okay, I'll close this.

**Comment by oscarbenjamin** [comment]
Looks like I merged this by mistake. I don't think it matters though so we can just leave it.

**Pull request merged**